src/components/Fruit.js

```diff
@@ -0,0 +1,31 @@
+import React, {useState} from "react";
+import Emoji from "react-emoji-render";
+
+function Fruit(props) {
+    const [count, setCount] = useState(0);
+    const handleTogglePlus = () => {
+        setCount(count + 1)
+    }
+    const handleToggleMin = () => {
+        if(count > 0) {
+            setCount(count - 1)
+        }
+    }
```

Comment:
Ik zie liever dat er door middel van een `disabled` property de button uitgezet wordt wanneer de counter `0` is. Dan hoef je ook geen aparte functie te maken die ervoor zorgt dat de counter niet ónder de 0 gaat